benchmarking/chain-sync/configuration/log-config-ci.yaml

```diff
@@ -20,11 +20,16 @@ setupScribes:
   - scKind: FileSK
     scName: "node-0.log"
     scFormat: ScText
```

Comment:
we can remove the "node-0.log" textual output.

Reply:
@CodiePP, that requires converting the rest to the structured logging -- will do.

Reply:
Done, pushed conversion to the structured logs.

Reply:
you just have overwritten my commit which allowed me to remove the node-0.log textual log.
I will force push again...